static/ejs/partials/components/pagesScannedModal.ejs

```diff
@@ -1,27 +1,31 @@
-<div 
-    id="pagesScannedModal"
-    class="modal fade"
-    aria-labelledby="pagesScannedModalLabel"
-    aria-hidden="true"
-    tabindex="-1"
-    >
-    <div class="modal-dialog">
-        <div class="modal-content">
-            <div class="modal-header">
-                <h4 class="modal-title fw-bold" id="pagesScannedModalLabel"><%= scanType === 'Customized' ? 'Custom Flow' : totalPagesScanned + ' pages'%></h4>
-                <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>  
-            </div>
-            <div class="modal-body">
-                <h6 class="modal-title fw-bold"><%= scanType === 'Customized' ? totalPagesScanned + ' pages' : ''%></h6>
-                <ul class="unbulleted-list">
-                    <% pagesScanned.forEach((page, index) => { %>
-                        <li>
-                          <a href="<%= page.url %>" target="_blank"><%= page.pageTitle %></a>
-                          <span><%= page.url %></span>
-                        </li>
-                    <% }) %>
-                </ul>
-            </div>
-        </div>
+<div
+  id="pagesScannedModal"
+  class="modal fade"
+  aria-labelledby="pagesScannedModalLabel"
+  aria-hidden="true"
+  tabindex="-1"
+>
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h4 class="modal-title fw-bold" id="pagesScannedModalLabel">
+          <%= scanType === 'Customized' ? 'Custom Flow' : totalPagesScanned + ' pages'%>
+        </h4>
+        <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+      </div>
+      <div class="modal-body">
+        <h6 class="modal-title fw-bold">
+          <%= scanType === 'Customized' ? totalPagesScanned + ' pages' : ''%>
+        </h6>
+        <ul class="unbulleted-list">
+          <% pagesScanned.forEach((page, index) => { %> <% if (page.actualUrl) { return; }%>
```

Comment:
cmiiw I think we can remove this line, since the logic now doesn't include already visited redirected sites into pagesScanned

Reply:
Yep, you're right! removed thanks!